client/src/services/index.js

```diff
@@ -62,6 +86,25 @@ export const getInvoice = async (id) => {
   return response.json();
 };
 
+export const createInvoice = async (data) => {
+  const response = await fetch('http://localhost:3000/api/invoices', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(data),
+  });
+  return response;
+};
+
+// *Booking Customer*
+export const getCustomersByBookingId = async (id) => {
+  const response = await fetch(
+    `http://localhost:3000/api/bookingcustomers?bookingId=${id}`
```

Comment:
I think you might be a little confused here.
- BookingId in param is auto generated in database (eg, 1, 2,...)
- As for 101, this means RoomId
- So the return value will be empty because there is no such big BookingId
![image](https://github.com/user-attachments/assets/c1e6902c-3bdc-4f4f-acf9-8e0a748ea3f1)

Anyway, thanks for checking the APIs so carefully and letting me know